content/blog/2021/announcing-in-kind-sponsorship.md

```diff
@@ -0,0 +1,42 @@
+Title:Call for In-Kind Sponsors for PyCon India 2021
+Date: 2021-06-06 17:00
+Category: 2021
+Slug: call-for-in-kind-sponsors
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India.
+
+
+As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices are also kept very minimal so that everyone can afford them. However, we are starting with In-kind Sponsorship this year so that Companies/Organizations who are still interested to contribute to the event and add value to the community can come forward and make the most out of this opportunity.
+
+## What are In-Kind Sponsors?
+
+In-kind sponsorship is a type of sponsorship where the sponsor agrees to provide swags (value-in-kind) instead of cash as part of a sponsorship agreement. These benefits/swags would be directly given to PyCon India 2021 attendees. In return for this value add/sponsorship, PyCon India 2021 team would be providing certain benefits to Sponsors, please refer to the Sponsor-Benefits section below. Regarding, what type of swags/benefits can be given to attendees, please refer to the guidelines below.
```

Comment:
```suggestion
In-kind sponsorship is a type of sponsorship where the sponsor agrees to provide swags (either physical or virtual, value-in-kind) instead of cash as part of the sponsorship agreement with PyCon India. These swags or benefits would be made directly available to all PyCon India 2021 attendees. In return for this value-added sponsorship, PyCon India 2021 team would be providing certain benefits to sponsors before, during, and after the event. 
```

Reply:
Yo @anistark we doing physical too? 
if not @Sc0rpi0n101 disregard that bit

Reply:
@anistark ^

Reply:
Yes. Can be some goodies or books, etc that they can send directly to the participants.

Reply:
Cool beans